pact.cabal

```diff
@@ -2,7 +2,7 @@
 -- see http://haskell.org/cabal/users-guide/
 
 name:                pact
-version:             2.3.7
+version:             2.3.8
```

Comment:
Let's hold off on upping the cabal as we need to do more release prep for the other changes in master. Should be able to get to it tonight. Thanks!